subsys/logging/log_core.c

```diff
@@ -485,6 +528,37 @@ u32_t log_filter_get(struct log_backend const *const backend,
 	}
 }
 
+char *log_strdup(char *str)
+{
+	size_t slen = strlen(str);
+	size_t cpylen;
+	char * strdup;
```

Comment:
It's probably wise to use some other symbol name (to avoid compiler/static checker warnings, etc), since a strdup function may be already provided by libc.

Reply:
strdup->duplicate